source/faq.md

```diff
@@ -80,8 +80,7 @@ If you have more than one monitor connected make sure you extend the screen inst
 ### Why am I seeing strange glitches in the video?
 
 Hardware acceleration is most likely not working correctly.
-Try going to **Options &rarr; Internal Filters**, right-click the list on the right
-and click "Disable DXVA filters" and then reload the video (<kbd>Ctrl+E</kbd> or manually).
+Try going to **Options &rarr; Internal Filters**, click **Video decoder**, set "Hardware decoder to use:" to "None" and then reload the video (<kbd>Ctrl+E</kbd> or manually).
```

Comment:
Please break this line before `, set` and adapt the remaining lines accordingly.